dist/src/main/resources/application-tasklist.properties

```diff
@@ -0,0 +1,43 @@
+spring.web.resources.add-mappings=true
+spring.web.resources.static-locations=classpath:/META-INF/resources/tasklist/
+# configure thymeleaf used by index.html
+spring.thymeleaf.check-template-location=true
+spring.thymeleaf.prefix=classpath:/META-INF/resources/tasklist/
+
+spring.mvc.problemdetails.enabled=false
+
+server.servlet.session.cookie.name=TASKLIST-SESSION
+# Return error messages for all endpoints by default, except for Internal API.
+# Internal API error handling is defined in InternalAPIErrorController.
+server.error.include-message=always
+
+management.health.defaults.enabled=false
+management.endpoint.health.probes.enabled=true
+management.endpoints.web.exposure.include=health,prometheus,loggers,usage-metrics,backups
+management.endpoint.health.group.readiness.include=readinessState,searchEngineCheck
+
+graphql.playground.enabled=false
+graphql.servlet.exception-handlers-enabled=true
+graphql.extended-scalars=DateTime
+graphql.schema-strategy=annotations
+graphql.annotations.base-package=io.camunda.tasklist
+graphql.annotations.always-prettify=false
+graphql.annotations.input-prefix=
+#---
+spring.config.activate.on-profile=identity-auth
+spring.security.oauth2.resourceserver.jwt.issuer-uri=${camunda.identity.issuer:${camunda.tasklist.identity.issuerUrl:${CAMUNDA_TASKLIST_IDENTITY_ISSUER_URL:}}}
+# Fallback Identity configuration for deprecated env variable naming
+camunda.identity.issuer=${camunda.tasklist.identity.issuerUrl:${CAMUNDA_TASKLIST_IDENTITY_ISSUER_URL:}}
+camunda.identity.issuerBackendUrl=${camunda.tasklist.identity.issuerBackendUrl:${CAMUNDA_TASKLIST_IDENTITY_ISSUER_BACKEND_URL:}}
+camunda.identity.clientId=${camunda.tasklist.identity.clientId:${CAMUNDA_TASKLIST_IDENTITY_CLIENT_ID:}}
+camunda.identity.clientSecret=${camunda.tasklist.identity.clientSecret:${CAMUNDA_TASKLIST_IDENTITY_CLIENT_SECRET:}}
+camunda.identity.audience=${camunda.tasklist.identity.audience:}
+camunda.identity.baseUrl=${camunda.tasklist.identity.baseUrl:}
+
+#---
+spring.config.activate.on-profile=sso-auth
+camunda.identity.type=AUTH0
+camunda.identity.baseUrl=${camunda.tasklist.identity.baseUrl:${CAMUNDA_TASKLIST_IDENTITY_BASEURL:}}
+camunda.identity.issuer=${camunda.tasklist.identity.issuerUrl:${CAMUNDA_TASKLIST_IDENTITY_ISSUER_URL:}}
+camunda.identity.issuerBackendUrl=${camunda.tasklist.identity.issuerBackendUrl:${CAMUNDA_TASKLIST_IDENTITY_ISSUER_BACKEND_URL:}}
+camunda.identity.audience=${camunda.tasklist.identity.audience:}
```

Comment:
You can delete this line (43), So far after analyse, it wont be used.

Reply:
Thanks, I will remove it

Reply:
Fixed